base/setup/lib/muifonts.h

```diff
@@ -126,6 +200,29 @@ MUI_SUBFONT HebrewFonts[] =
     { L"Times New Roman Greek,161",    L"Times New Roman,161" },
     { L"Times New Roman TUR,162",      L"Times New Roman,162" },
     { L"Tms Rmn",                L"Times New Roman" },
+    /* The beginning of FE Asian fonts */
```

Comment:
I wonder why these are necessary in non-asian lists, like hebrew, hindi, etc.

Reply:
FontLink framework will realize the mix of various fonts in one text. If the font is not available, FontLink won't work.

Reply:
Ok, I will make a separated list.

Reply:
― By the power of FontLink, ...
Latin users will be able to see Asian characters.
Asian users will be able to see Latin accented characters.

Reply:
I understand you don't like Asian complex. But FontLink is the only one way to realize real international Windows operating system.

Reply:
Is fontlinking available in 2k3sp2?
And if yes, how does MS do it?

Reply:
English Win2k3 doesn't have Asian fonts and FontLink in default settings. There is additional package for Far East Asian.
![Win2k3-English-doesnt-have-FontLink](https://github.com/reactos/reactos/assets/2107452/7e5fcdd1-2c06-4944-928d-6a1b9663e07a)

Japanese WinXP has Chinese, Japanese, and Korean fonts and Font Link.
![WinXP-Japanese-has-FontLink-and-Chinese-fonts](https://github.com/reactos/reactos/assets/2107452/946c67b2-ec70-4fe2-b66b-dc5ebdc370e4)

Chinese WinXP has Chinese, Japanese, and Korean fonts and Font Link.
![WinXP-Chinese-has-FontLink-and-Japanese-fonts](https://github.com/reactos/reactos/assets/2107452/7f996965-d1db-4516-b46e-aba69f870907)

Reply:
So, I think that, FontLink is initially provided for East Asian people.
English and Latin users don't want FontLink so much.

Reply:
Fixed.